src/config-manager.ts

```diff
@@ -1,8 +1,9 @@
 import fs from 'fs-extra';
 import mongoose from 'mongoose';
 import dotenv from 'dotenv';
-import { LOG_INFO, LOG_WARN, LOG_ERROR } from '@/logger';
-import { Config } from '@/types/common/config';
+import { LOG_INFO, LOG_WARN, LOG_ERROR, formatHostnames } from '@/logger';
+import { Config, domainServices } from '@/types/common/config';
+import isValidHostname from 'is-valid-hostname';
```

Comment:
We have a standard import order for imports, which goes:

- Built in modules (using the `runtime:` prefix)
- 3rd party modules
- Local modules
- Types for 3rd party modules (using `import type`)
- Types for local modules (using `import type`)
- Data file imports (such as importing JSON files)

The only real exception to this is in the router files, where after the standard module imports we add a blank newline and then import all the routes in alphabetical order

Reply:
Noted, I will correct this

You will probably want to look into the eslint rule [import/order](https://github.com/import-js/eslint-plugin-import/blob/main/docs%2Frules%2Forder.md) to allow people to lint and fix this as they go along

Reply:
I had no idea that was a thing, will definitely look into it tomorrow